src/ray/raylet/node_manager.cc

```diff
@@ -975,6 +980,8 @@ void NodeManager::HeartbeatAdded(const ClientID &client_id,
   }
 
   if (new_scheduler_enabled_ && client_id != self_node_id_) {
+    // TODO (Alex): Don't forget to implement this.
```

Comment:
Minor, but I don't think the comment is necessary haha.

Reply:
lol, if it's not too much trouble for everyone else, I would prefer to leave it here (easier to look for)

Reply:
Can you make it more descriptive? Like "Implement light heartbeats" or something.